qlib/Logger.qm

```diff
@@ -38,9 +38,6 @@ module Logger {
     author = "Tomas Mandys <tomas.mandys@qore.org>";
```

Comment:
Please fix qore.com vs. qoretechnologies.com copy paste error I introduced somewhere in the past